ui/src/join_game/TimerComponent.tsx

```diff
@@ -19,30 +21,38 @@ const TimerComponent = () => {
   });
 
   const recalculateTimer = () => {
+    if (timer_runs_out === 0) {
+      return;
+    }
     set_seconds_left(timer_runs_out - Math.floor(Date.now() / 1000));
   };
 
-  const getSecondsLeftString = (runs_out: Number, seconds_left: Number) => {
-    if (runs_out === 0) {
-      return "Not started yet!";
-    } else if (seconds_left <= -1000000) {
-      return 300;
-    } else {
-      if (seconds_left <= 0) {
-        return "Ran out of time!";
-      } else {
-        return seconds_left;
-      }
+  const getMessage = (): string => {
+    let defeat_message = "Defeat! You ran out of time!";
```

Comment:
y does this get it's own variable and no one else does

Reply:
Because I gotta use it twice.